packages/@aws-cdk/aws-apigatewayv2/lib/websocket/api.ts

```diff
@@ -26,6 +41,12 @@ export interface WebSocketApiProps {
    */
   readonly apiName?: string;
 
+  /**
+   * An API key selection expression. Currently only supports '$request.header.x-api-key' and '$context.authorizer.usageIdentifierKey'
```

Comment:
Same here. Think of a user who is coming to the cdk and looking at these. They should be able to understand at a basic level what this is, whether they should use it, etc.